crates/common/src/zk_compile.rs

```diff
@@ -297,7 +298,12 @@ impl ZkSolc {
         let mut displayed_warnings = HashSet::new();
         let mut data = BTreeMap::new();
         // Step 1: Collect Source Files
-        let sources = self.get_versioned_sources().wrap_err("Cannot get source files")?;
```

Comment:
Needed to replace `wrap_err` being used everywhere because it was making it impossible for debugging 🤦